src/search.cpp

```diff
@@ -923,7 +923,8 @@ namespace {
               continue;
 
           // History based pruning
-          if (   depth <= 3 * ONE_PLY
+          if (   depth <= 4 * ONE_PLY
+              &&  move != ss->killers[0]
```

Comment:
formatting: i think there should be only one space before "move" aligned left with depth on previous line